### DISCUSSION TITLE
Enlace a grafana en producción?

### DISCUSSION THREAD

**Discussion by Santiago21112001** [Arquisoft/faq#72] [Q&A] [answered]
Buenas somos wiq_es04d. Tenemos activado grafana y prometheus porque no hemos cambiado nada de la plantilla inicial. ¿Hay que poner algun enlace a los dashboards de grafana en la app en producción? ¿Cómo demostramos que la app está siendo monitorizada?

**Comment by labra** [comment, marked as answer]
Podéis hacer varias opciones:
- Incluir el enlace en la aplicación como tú dices, aunque igual no queda muy elegante para un usuario normal que pueda acceder a esos datos
- Incluir un enlace en el repositorio github
- Incluir un enlace en la documentación

En mi opinión cualquiera de esas opciones puede servir. Vamos, lo importante es que esté visible para los profesores y que lo encontremos facilmente.
